iothub/device/src/Transport/AmqpIoT/AmqpIoTErrorAdapter.cs

```diff
@@ -30,14 +30,6 @@ internal static class AmqpIoTErrorAdapter
         public static readonly AmqpSymbol PartitionNotFound = AmqpIoTConstants.Vendor + ":partition-not-found";
         public static readonly AmqpSymbol IotHubSuspended = AmqpIoTConstants.Vendor + ":iot-hub-suspended";
 
-        public static readonly AmqpSymbol TrackingId = AmqpIoTConstants.Vendor + ":tracking-id";
-        public static readonly AmqpSymbol ClientVersion = AmqpIoTConstants.Vendor + ":client-version";
-        public static readonly AmqpSymbol ApiVersion = AmqpIoTConstants.Vendor + ":api-version";
-        public static readonly AmqpSymbol ChannelCorrelationId = AmqpIoTConstants.Vendor + ":channel-correlation-id";
-        public static readonly AmqpSymbol AuthChain = AmqpIoTConstants.Vendor + ":auth-chain";
-
-        private const int MaxSizeInInfoMap = 32 * 1024;
-
```

Comment:
is line 39 not needed anymore? if so why?

Reply:
This was required in one of our older "convert amqp outcome to iot hub exception" methods. That method is no longer in use, so this variable was simply lying around, and not being referenced anywhere.

Reply:
We have updated implementation for "convert amqp outcome to iot hub exception" now, one that no longer uses this `MaxSizeInInfoMap`.